sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/init.sql

```diff
@@ -0,0 +1,61 @@
+CREATE OR REPLACE TABLE
+  `mozilla-public-data`.telemetry_derived.agg_active_users_v1
+PARTITION BY
```

Comment:
Should we also define cluster fields here?

Reply:
Makes sense! Added country, app_name, attribution_medium, channel.

Reply:
cool, can you specify the same ones here:

https://github.com/mozilla/bigquery-etl/pull/2952/files#diff-2e37bcde22da96b36b063787a0b603e21093ec0a23a11f2657744b0e530a9f99R21-R24

Reply:
Should be there already..

Reply:
Ideally, we'd want to avoid using `init.sql` and instead create the tables through `schema.yaml` files: https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#adding-a-new-scheduled-query

You should be able to generate the schema files via: `./bqetl query schema update <dataset>.<table>_<version>`
And then create the BigQuery table: `./bqetl query schema deploy`
Partitioning and clustering information can be added to the `metadata.yaml` file.

Reply:
Thanks for this comment Anna, I notice two different approaches in the documentation: one for using init.sql file and one for using schema deploy. Shall we unify to a standard one? Please see this [docs update PR](https://github.com/mozilla/bigquery-etl/pull/2960)?